README.md

```diff
@@ -1,273 +1,247 @@
+// Result: Inconsistent data, broken dashboards
+```
 
-- Built with Rust, Weaver offers performance comparable to C or C++ implementation.
-  The entire OpenTelemetry semantic convention registry can be downloaded, resolved,
-  and documented in under 2 seconds.
-- Semantic Convention Registry Git repositories can be efficiently cached locally.
-- Registry and schema validation, as well as documentation and code generation,
-  are parallelized for optimal performance.
+**With Weaver:**
 
-## Robustness and Reliability
+```rust
+// Both developers:
+counter.add(1, [(attributes::HttpMethod::KEY, "GET")]);
+// Result: Consistent data, reliable observability
+```
 
-- **Memory Safety**: Rust ensures memory safety, preventing common vulnerabilities.
-- **Comprehensive Error Reporting**: Weaver reports as many errors as possible in
-  a single pass, providing developers with comprehensive feedback.
-- **Quality Assurance**: Code coverage, Cargo deny, Dependabot, and automated security
-  audits.
+## Install
 
-## Cross-Platform Compatibility
+**Pre-built binaries:**
 
-- Tested Platforms: Weaver is manually tested on Linux, macOS, and Windows.
-- Future Plans: Automated tests will be implemented for broader platform coverage.
+Linux, Windows and Mac installers on the [releases](https://github.com/open-telemetry/weaver/releases) page.
 
-## Getting started
+**Docker:**
 
-Currently, weaver can be consumed in one of three ways:
+```bash
+docker pull otel/weaver
+```
 
-- [Pre-built Binaries](#pre-built-binaries)
-- [Docker Image](#docker-image)
-- [Building from source](#building-from-source)
+**From source:**
 
-### Pre-built Binaries
+```bash
+git clone https://github.com/open-telemetry/weaver.git
+cd weaver
+cargo build --release
+```
 
-Weaver release attach pre-built binaries for supported platforms with every release.
-Instructions for installing are included in these release notes.
+## Define Your Schema
 
-See: [Weaver Releases](https://github.com/open-telemetry/weaver/releases)
+![Observability by Design Workflow - Define](docs/images/workflow_define.svg)
 
-### Docker Image
+Weaver leverages the [OpenTelemetry Semantic Conventions](https://opentelemetry.io/docs/specs/semconv/) - 900+ standardized attributes across 74 domains, maintained by expert groups.
 
-Weaver deploys a docker image for development to [docker hub](https://hub.docker.com/r/otel/weaver).
+Use official conventions, extend them, or create custom registries for your needs.
 
-Instructions for using the docker image can be found [here](docs/docker-guide.md).
+Create `my-app.yaml`:
 
-### Building from source
+```yaml
+groups:
+  - id: span.example_message
+    type: span
+    stability: development
+    brief: This span represents a simple message.
+    span_kind: client
+    attributes:
+      - ref: example.message
+        requirement_level: required
+      - ref: host.name
+        requirement_level: required
+      - ref: host.arch
+        requirement_level: required
+```
```

Comment:
After this yaml file, we could talk about the custom registry declaration itself and introduce the `registry_manifest.yaml` file.